housing_cooperative_base/models/cluster.py

```diff
@@ -17,34 +17,34 @@ class Cluster(models.Model):
     )
     code = fields.Char(string="Code", required=True)
     keys = fields.Char(string="Keys", required=False)
-    housing_ids = fields.One2many(
-        comodel_name="hc.housing",
-        inverse_name="cluster_id",
-        string="Housings",
-        required=False,
-    )
-    room_ids = fields.One2many(
-        comodel_name="hc.room",
-        inverse_name="cluster_id",
-        string="Common Rooms",
-        required=False,
-    )
-    cellar_ids = fields.One2many(
-        comodel_name="hc.cellar",
-        inverse_name="cluster_id",
-        string="Cellars",
-        required=False,
-    )
+    # housing_ids = fields.One2many(
```

Comment:
Can we clean this up now?

Reply:
It's possible we could get rid of the entire concept of 'cluster', or will modify their specs. Tbd with the clients.

Reply:
Do I take the concept of 'cluster' out of this branch and put it in another branch, or do we leave this as such? Right now you can make clusters but not add anything to them or use them in a lease.

Reply:
If we end up not using them, I'd rather remove it altogether to avoid stale code. But we can wait until the client decides. :-)